app/views/layouts/_navigation.html.erb

```diff
@@ -12,9 +12,9 @@
     <ul class="nav navbar-nav navbar-right">
       <form class="navbar-form navbar-left" role="search" class="search_form" style="margin-top:6px;margin-bottom:4px;">
         <div class="form-group">
-          <input name ="q" type="text" class="form-control input-sm" placeholder="Пошук" size="30"/>
+          <%= text_field_tag 'q', nil, size: 30, class: 'form-control input-sm', value: params[:q] %>
         </div>
-        <input type="submit" class="btn btn-default btn-sm" value="Пошук" />
+        <%= submit_tag t('button') ,  class: "btn btn-default btn-sm", name: nil  %>
```

Comment:
Please give that button appropriate name. There will be a lot of buttons. Maybe something like `t('button_search_top')` is good.

Reply:
I think just `search` is ok for the translation key.
Also please remove space before comma, and live only one space before `%>`